src/app/features/event/card/card.component.html

```diff
@@ -0,0 +1,36 @@
+<div
+    *ngIf="card"
+    [style.background-image]="'url(' + this.url + ')'"
+    [ngClass]="[cardSize, themeColor]"
+    class="card">
+    <app-power-icon
+        class="card__power-icon"
+        [memberLimit]="card.memberLimit"
+        [memberNumber]="card.invitedMembersNumber"
+        [size]="'sm'"
+        [theme]="iconTheme"></app-power-icon>
+    <div class="card__main-photo">
+        <img src="{{ card.mainPhoto }}" alt="" />
+    </div>
+    <div class="card__info" [ngClass]="{ 'text-body-sm': size === 'sm' }">
```

Comment:
jak mamy jedną klasę to może odnieść się bezpośrednio do bindingu danej klasy czyli

`[ngClass]="{ 'text-body-sm': size === 'sm' }"`

można zapisać jako `[class.text-body-sm]="size === 'sm'"`

wybór zostawiam wam

Reply:
Dzięki za porade. Twój sposób zapisu bardziej mi się podoba, zamieniłem swój na twój.